tests/sals/vdc/vdc_base.py

```diff
@@ -66,22 +66,23 @@ def _start_threebot_server(cls):
         cls.server.start()
 
     @classmethod
-    def deploy_vdc(cls):
+    def deploy_vdc(cls, hours=1):
         cls.vdc_name = cls.random_name().lower()
         cls.password = cls.random_string()
         cls.vdc = j.sals.vdc.new(cls.vdc_name, cls.tname, cls.flavor)
 
-        cls.info("Transfer needed TFT to deploy vdc for an hour to the provisioning wallet.")
+        cls.info(f"Transfer needed TFT to deploy vdc for {hours} hour/s to the provisioning wallet.")
         cls.vdc_price = j.tools.zos.consumption.calculate_vdc_price(cls.flavor)
-        needed_tft = (
+        needed_tft = hours * (
             float(cls.vdc_price) / 24 / 30 + 2 * TRANSACTION_FEES
         )  # 2 transaction fees for creating the pool and extend it
```

Comment:
we need here only on transaction fee for extend.
```suggestion
        extension_fees = TRANSACTION_FEES if hours > 1 else 0
        needed_tft = hours * (
            float(cls.vdc_price) / 24 / 30 ) + 2 * TRANSACTION_FEES + extension_fees
        )  # 2 transaction fees for creating the pool and extend it
```